### DISCUSSION TITLE
What's the "Boot Delay" for?

### DISCUSSION THREAD

**Discussion by aicayzer** [snezhig/obsidian-front-matter-title#207] [Q&A]
Hey, 

As the title says, what's the "Boot Delay" for? i.e. by setting it to <1000ms, is there a risk of an issue?

Thanks :)

**Comment by snezhig** [comment]
Sometimes errors occur related to loading the application or third-party plugins. To avoid this, you can set the boot delay to an arbitrary value. You can try setting it to 0 and see if you have problems.

**Reply by aicayzer** [reply]
Makes sense, thank you :)
